apps/nextjs/src/app/[locale]/manage/users/groups/[id]/_delete-group.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import { useCallback } from "react";
+import { useRouter } from "next/navigation";
+import { Button } from "@mantine/core";
+
+import { clientApi } from "@homarr/api/client";
+import { useConfirmModal } from "@homarr/modals";
+import {
+  showErrorNotification,
+  showSuccessNotification,
+} from "@homarr/notifications";
+import { useI18n, useScopedI18n } from "@homarr/translation/client";
+
+import { revalidatePathAction } from "~/app/revalidatePathAction";
+
+interface DeleteGroupProps {
+  group: {
+    id: string;
+    name: string;
+  };
+}
+
+export const DeleteGroup = ({ group }: DeleteGroupProps) => {
+  const router = useRouter();
+  const { mutateAsync } = clientApi.group.deleteGroup.useMutation();
+  const { openConfirmModal } = useConfirmModal();
+  const tDelete = useScopedI18n("group.action.delete");
+  const t = useI18n();
```

Comment:
I dislike that the above translation hook is named and this one is not. It's unclear what quantity / scope this is